Makefile

```diff
@@ -5,6 +5,7 @@ else
 endif
 
 CARGO ?= cargo
+CARGOFLAGS += --locked
```

Comment:
I think the `Makefile.Windows` and `CMakeLists.txt` files might need similar updates.

Reply:
Done